docker/openemr/7.0.2/openemr.sh

```diff
@@ -24,8 +24,26 @@ swarm_wait() {
 auto_setup() {
     prepareVariables
 
-    chmod -R 600 .
```

Comment:
hi @Jmevorach ,
Was about to bring this in, but noted the above line was removed. Was removing above line needed for below changes to work?

Reply:
Hello @bradymiller! Good catch; apologies for that that was a mistake. Didn't mean to take that out. I put it back in and tested again for 7.0.2!